packages/server/src/api/routes/rowAction.ts

```diff
@@ -50,6 +50,16 @@ router
     authorizedResource(PermissionType.TABLE, PermissionLevel.READ, "tableId"),
     rowActionController.remove
   )
+  .post(
+    "/api/tables/:tableId/actions/:actionId/permissions/:viewId",
+    authorizedResource(PermissionType.TABLE, PermissionLevel.READ, "tableId"),
```

Comment:
These endpoints are only used for setting the view permissions - I think these should probably only be authorized as builder actions - these should never be called by real users in the end app.